addons/colored-follower-projects/userscript.js

```diff
@@ -0,0 +1,47 @@
+//Hello! I've been scripting for a while now, but i'm not great at keeping my code clean. I'll try to make it readable, but please excuse my mistakes!
+export default async function ({ addon, global, console }) {
+    if(addon.auth.isLoggedIn){ // If they are logged in, activate the rest of the code
+        var username = "griffpatch" // Default username
+        if(addon.settings.get("useother")){ // If using "other scratchers following"
+            username = addon.settings.get("useothername") // Set the username to the one provided
+        }else{
+            username = addon.auth.username // Not using "other scratchers following"? Just set it to your username.
+        }
```

Comment:
```js
const username =  addon.settings.get("useothername") || addon.auth.username
```

If a username is specified use it, otherwise use the current logged in user.

Then you should remove the checkbox setting.